File: Loading/QLS2/src/qls/ast/widget/Radio.java
package qls.ast.widget;

public class Radio extends Widget {
	
	private final String checked;
	private final String unchecked;
	
	Radio(String checked, String unchecked, int line) {
		super(line);
		this.checked = checked;
		this.unchecked = unchecked;
	}
}

File: WeekendWarriors/Questionnaires/Questionnaires/QLS/AST/Widgets/Radio.cs
using Questionnaires.UI.Widgets;
using Questionnaires.QL.AST.Types;

namespace Questionnaires.QLS.AST.Widgets
{
    public class Radio : Widget
    {
        public override QuestionWidget CreateWidget(BooleanType type)
        {
            return new RadioWidget();
        }
    }
}
